### PR TITLE
development doc updates

### PR DESCRIPTION
**Pull request by SethTisue**
especially Google Groups -> Discourse, but also 2.11->2.12 and other
misc

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,34 +1,38 @@
...
+You're always welcome to submit your PR straight away and start the discussion (without reading the rest of this wonderful doc, or the [`README.md`](README.md)). The goal of these notes is to make your experience contributing to Scala as smooth and pleasant as possible. We're happy to guide you through the process once you've submitted your PR.
 
 ## The Scala Community
+
 In 2014, you -- the Scala community -- matched the core team at EPFL in number of commits contributed to Scala 2.11, doubling the percentage of commits from outside EPFL/Lightbend since 2.10. Excellent work! (The split was roughly 25/25/50 for you/EPFL/Lightbend.)
 
 We are super happy about this, and are eager to make your experience contributing to Scala productive and satisfying, so that we can keep up this growth. We can't do this alone (nor do we want to)!
 
-This is why we're collecting these notes on how to contribute, and we hope you'll share your experience to improve the process for the next contributor! (Feel free to send a PR for this note, send your thoughts to gitter, scala-internals, or tweet about it to @adriaanm.)
```
Comment (adriaanm) [inline CONTRIBUTING.md:11]: use links? 
  - the [scala/contributors Gitter](https://gitter.im/scala/contributors)
  - the [contributors Discourse](https://contributors.scala-lang.org/c/documentation)

**Comment by adriaanm** [comment]
Thanks, one minor nitpick bug good to merge as-is. Oh, one follow-up: could you see about reflecting our new policy on ticket references in commit messages (i.e., title is only description, PR body has "Fixes $ref")

**Review by lrytz — APPROVED** [review]
👍

File: CONTRIBUTING.md (hunk 1 above)
Comment (SethTisue) [reply CONTRIBUTING.md:11]: (meh, the links already appear earlier in the file)

**Pull request merged**